astroimtools/cutout_tools.py

```diff
@@ -16,11 +16,270 @@
 from astropy import log
 from astropy.coordinates import SkyCoord
 from astropy.io import fits
+from astropy.io.fits import PrimaryHDU, ImageHDU, CompImageHDU
 from astropy.nddata.utils import (Cutout2D, NoOverlapError)
-from astropy.table import QTable
+from astropy.table import QTable, Table
 from astropy.wcs import WCS, NoConvergence
+from astropy.wcs.utils import proj_plane_pixel_scales
 
-__all__ = ['make_cutouts', 'show_cutout_with_slit']
+__all__ = ['make_cutouts', 'show_cutout_with_slit', 'cutout_tool']
+
+
+def cutout_tool(image, catalog, image_ext=0, origin=0,
+                to_fits=False, output_dir="cutouts", overwrite=False,
+                catalog_format='ascii.ecsv', delimiter=None,
+                suppress_rotation=False, verbose=True):
+    """
+    Make cutouts from a 2D image and write them to FITS files.
+
+    The input Catalog must have the following columns with unit info, where applicable:
```

Comment:
(minor) maybe just be a bit more specific here, saying something like ``... following columns, which must have `~astropy.unit.Unit`s where applicable.`` - that way it's clear that you mean they need units in the sense that Astropy tables can have units, rather than just meaning "some information about units".